test/v2/models/request/amendHistoricFhlUkPiePeriodSummary/AmendHistoricFhlUkPiePeriodSummaryRequestBodySpec.scala

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.request.amendHistoricFhlUkPiePeriodSummary
+
+import play.api.libs.json.{ JsValue, Json }
+import support.UnitSpec
+import v2.models.request.common.ukFhlPieProperty.{ UkFhlPieExpenses, UkFhlPieIncome }
+import v2.models.request.common.ukPropertyRentARoom.{ UkPropertyExpensesRentARoom, UkPropertyIncomeRentARoom }
+
+class AmendHistoricFhlUkPiePeriodSummaryRequestBodySpec extends UnitSpec {
+
+  val income: UkFhlPieIncome = UkFhlPieIncome(
+    Some(100.25),
+    Some(100.15),
+    Some(UkPropertyIncomeRentARoom(Some(97.50)))
+  )
+
+  val expenses: UkFhlPieExpenses = UkFhlPieExpenses(Some(123.12),
```

Comment:
You might want to use named arguments for these long arg lists to aid comparison when reading the tests.